src/components/tab/SearchInput.vue

```diff
@@ -130,6 +130,19 @@ export default {
 			}
 			return ''
 		},
+		getNoOptionText() {
+			switch (this.noOptionTextState) {
+			case NO_OPTION_TEXT_STATE.START_TYPING :
+				return t('integration_openproject', 'Start typing to search')
+			case NO_OPTION_TEXT_STATE.RESULT :
+				if (this.searchResults.length === 0) {
+					return t('integration_openproject', 'There were no workpackages found')
+				}
+				break
+			}
+			// while workpackages are being searched we make the no text option empty
+			return t('integration_openproject', '')
```

Comment:
```suggestion
			return  ''
```
I think no need to translate the empty string